vendor/modules.txt

```diff
@@ -2406,6 +2406,7 @@ k8s.io/metrics/pkg/client/custom_metrics/fake
 k8s.io/metrics/pkg/client/custom_metrics/scheme
 k8s.io/metrics/pkg/client/external_metrics
 k8s.io/metrics/pkg/client/external_metrics/fake
+# k8s.io/mount-utils => ./staging/src/k8s.io/mount-utils
```

Comment:
Why is this commented out?

Reply:
Oh maybe it's because nothing in Kubernetes is using it yet? We may have to remove the line, I think it is causing the latest dependencies run to fail

Reply:
I agree, we can remove this for now. I am trying to understand 'go vendor' and k/k tooling on how it formats this file. I worry about this when we enable new repo.

Reply:
This commented line is required for verify-vendor to be happy. We have more work in the subsequent PRs to remove utils/mount completely from code and I will address further changes to vendor/modules then.